ui/app/components/clients/page/counts.hbs

```diff
@@ -12,8 +12,32 @@
     @upgradesDuringActivity={{this.upgradesDuringActivity}}
     @noData={{not @activity.total.clients}}
     @onChange={{this.onDateChange}}
+    @showEditModal={{this.showEditModal}}
+    @setEditModalVisible={{this.setEditModalVisible}}
   />
 </div>
+{{#if (and this.version.isCommunity this.formattedStartDate this.formattedEndDate)}}
```

Comment:
it looks like this conditional is used in quite a few places. I wonder if it could be consolidated into a getter somewhere and reused to be something like `shouldDisplayCommunityVersion` or something